### ISSUE TITLE
Problem Analyzing Classes

### ISSUE THREAD

**Issue by AdamBien** [sdaschner/jaxrs-analyzer-maven-plugin#4]
Hi Sebastian,

I tried the analyzer in my stateful project and it has some trouble analyzing some classes. I separated the JAR and WAR and now the analyzer omits some resources.

Reproducer: 

git clone https://github.com/AdamBien/stateful
mvn clean install

The main project is:
cd statefulapp
mvn clean install

Logs---

analyzing JAX-RS resources, using Swagger backend
Could not analyze the method: javassist.CtMethod@5899a62[public all ()Ljavax/json/JsonArray;]
Could not interpret project method: {className='com.airhacks.stateful.machine.control.SCXMLExecutorFactory', methodName='create', returnType='org.apache.commons.scxml2.SCXMLExecutor', staticMethod='true', parameterTypes=[org.apache.commons.scxml2.model.SCXML]}
Could not interpret project method: {className='com.airhacks.stateful.machine.control.StateMachineStore', methodName='store', returnType='null', staticMethod='false', parameterTypes=[java.lang.String, org.apache.commons.scxml2.SCXMLExecutor]}
Could not interpret project method: {className='com.airhacks.stateful.machine.control.SCXMLExecutorFactory', methodName='create', returnType='org.apache.commons.scxml2.SCXMLExecutor', staticMethod='true', parameterTypes=[org.apache.commons.scxml2.model.SCXML]}
Could not interpret project method: {className='com.airhacks.stateful.machine.control.StateMachineStore', methodName='store', returnType='null', staticMethod='false', parameterTypes=[java.lang.String, org.apache.commons.scxml2.SCXMLExecutor]}
Could not analyze the method: javassist.CtMethod@95f86c56[public define (Ljava/lang/String;Ljava/io/InputStream;Ljavax/ws/rs/core/UriInfo;)Ljavax/ws/rs/core/Response;]
Could not analyze the method: javassist.CtMethod@8d045506[public reset (Ljava/lang/String;)Ljavax/ws/rs/core/Response;]
Could not analyze the method: javassist.CtMethod@b75d8b74[public transition (Ljava/lang/String;Ljavax/json/JsonObject;)Ljavax/ws/rs/core/Response;]
Could not analyze the method: javassist.CtMethod@1ff35829[public status (Ljava/lang/String;)Ljavax/ws/rs/core/Response;]
Could not analyze the method: javassist.CtMethod@c8917d8d[public get (Ljava/lang/String;)Ljavax/ws/rs/core/Response;]
Could not analyze the method: javassist.CtMethod@52c70dc5[public transition (Ljava/lang/String;Ljavax/ws/rs/core/UriInfo;Ljavax/json/JsonObject;)Ljavax/ws/rs/core/Response;]
## Could not analyze the method: javassist.CtMethod@1ff35829[public status (Ljava/lang/String;)Ljavax/ws/rs/core/Response;]

thanks in advance! (deployedDomain is nice),

--adam

**Comment by sdaschner** [comment]
Hi Adam,

thanks for issuing that!
Fixed in sdaschner/jaxrs-analyzer@e5592cb. Currently it is only available in the latest snapshot version `0.6-SNAPHOST` for which the Maven plugin snapshot repository has to be added:

```
<pluginRepositories>
    <pluginRepository>
        <id>ossrh-snapshots</id>
        <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
        <releases>
            <enabled>false</enabled>
        </releases>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </pluginRepository>
</pluginRepositories>
```

Anyway, I plan to release the 0.6 version soon, but fist I want to include some other fixes.

**Comment by sdaschner** [comment]
FYI: I've just released version `0.6` which includes that fix.

**Comment by AdamBien** [comment]
Thanks!
